template.html

```diff
@@ -38,7 +38,7 @@ <h1 class="title">{{ config.title }}</h1>
 
         <!-- {% if not panels %}
         <div class="panel operational" data-l10n-id="systems-operational">
-          All Systems Operational
+          All Systems Mostly Operational
```

Comment:
I feel like this is akin to "mostly harmless"

Reply:
Happy to remove! It is just weird because we don't update that.

Reply:
I didn't mean remove, I just thought the parallels with hitchhikers guide were funny

Reply:
ahhhh putting it back!